### DISCUSSION TITLE
No preview on localhost

### DISCUSSION THREAD

**Discussion by Ruairi-azyra** [badges/shields#6902] [Q&A]
The intention of the project I am working on is to display shieldsio badges on the Readme of a private GitHub repository. 

Firstly, I got the badges working using the shields.io website for the public repo. Then moved on to try to self host the images and access them through the public repo.

To try make this work I published the images to a local container on Docker. The container was set to port 8080:80 and could be accessed at localhost. However when navigating to the shieldsio page at http:\\localhost:80 the preview of the badge would not appear, indicating that the badge was not reading the information, as shown below.

![image](https://user-images.githubusercontent.com/87992931/129581141-f1c6dd21-649b-4eb9-9096-61b79f81b86f.png)

Is there any additional configuration steps not in the documentation required to set up shields.io locally?

Just to note:
I have also tried following server-secrets.md, by adding the docker hub key as the value of a repo secret and trying to read the container from a GitHub workflow but I believe this is not the issue.

**Comment by chris48s** [comment]
Have you looked at the docs for running shields locally with Docker?
https://github.com/badges/shields/blob/master/doc/self-hosting.md#docker
As far as I know they are complete/up-to-date
